### PR TITLE
fix - tck contains is invalid

### PR DESCRIPTION
**Pull request by heroicNeZha**
#### What type of PR is this?
- [x] bug
- [ ] feature
- [ ] enhancement

#### What does this PR do?
fix bug : tck contains is invalid
    When executing query:
      """
      SHOW USERS
      """
    Then the result should contain:
      | Account |
      | "xxxxx"  |
get passed

#### Checklist：
- [ ] Documentation affected （Please add the label if documentation needs to be modified.)
- [ ] Incompatible （If it is incompatible, please describe it and add corresponding label.）
- [ ] Need to cherry-pick （If need to cherry-pick to some branches, please label the destination version(s).)
- [ ] Performance impacted: Consumes more CPU/Memory

#### Release notes：
Please confirm whether to reflect in release notes and how to describe:
>                                                                 `

### CODE REVIEW

**Comment by Aiee** [comment]
Please add some test cases.

**Comment by heroicNeZha** [comment]
> Please add some test cases.

**Comment by jievince** [comment]
> [CLA assistant check](https://cla-assistant.io/vesoft-inc/nebula?pullRequest=3314) Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/vesoft-inc/nebula?pullRequest=3314) before we can accept your contribution.You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/vesoft-inc/nebula?pullRequest=3314) it.

Please sign the CLA first

**Review by jievince — APPROVED** [review]
LGTM

**Review by CPWstatic — APPROVED** [review]

File: tests/tck/features/admin/Hosts.feature [hunk 1]
```diff
@@ -17,21 +18,21 @@ Feature: Admin hosts
       """
     Then the result should contain:
       | Host  | Port  | Status   | Role    | Git Info Sha  | Version |
-      | /\w+/ | /\d+/ | "ONLINE" | "GRAPH" | /[0-9a-f]{7}/ | EMPTY   |
+      | /\w+/ | /\d+/ | "ONLINE" | "GRAPH" | /[0-9a-f]{7}/ | ''      |
```
Comment (HarrisChu) [inline tests/tck/features/admin/Hosts.feature:21]: How about use /.*/ to match all?
I would run tck cases in release package.

**Review by jievince — DISMISSED** [review]

File: tests/common/comparator.py [hunk 2]
```diff
@@ -49,6 +49,7 @@ def s(self, b: bytes) -> str:
 
     def _whether_return(self, cmp: bool) -> bool:
         return ((self._contains == CmpType.EQUAL and not cmp)
+                or (self._contains == CmpType.CONTAINS and not cmp)
```
Comment (yixinglu) [inline tests/common/comparator.py:52]: I think it's not a right fix: contains means that response results should contains the user expected results. So there maybe exist some rows in response result set but not in expected result set. we could not return directly when we find some invalid rows. We should iterate the whole response result set and compare each row in expected result set with response row.

**Review by yixinglu — COMMENTED** [review]
you can use following commands to restore the unnecessary changes:

```bash
$ git reset HEAD~6
# restore changes
$ git commit -am '.....'
```

**Review by yixinglu — APPROVED** [review]
excellent

**Review by jievince — APPROVED** [review]
Good job!

**Pull request merged**